### ISSUE TITLE
Banner has infinite loop in multi-module project

### ISSUE THREAD

**Issue by robertpatrick** [kordamp/oci-maven-plugin#6]
The plugin seems to work find from a simple, single module project.  However, when I try to use it in my multi-module project, the goals never execute.  With debug on, I can see the configuration step runs and has all of the expected values:

```
...
[DEBUG] Configuring mojo org.kordamp.maven:oci-maven-plugin:0.5.0:put-object from plugin realm ClassRealm[plugin>org.kordamp.maven:oci-maven-plugin:0.5.0, parent: sun.misc.Launcher$AppClassLoader@4e25154f]
[DEBUG] Configuring mojo 'org.kordamp.maven:oci-maven-plugin:0.5.0:put-object' with basic configurator -->
[DEBUG]   (f) mojoExecution = org.kordamp.maven:oci-maven-plugin:0.5.0:put-object {execution: upload-online-file}
[DEBUG]   (s) bucketName = wdt-alias-test-files
[DEBUG]   (s) file = /Users/rpatrick/Projects/weblogic-deploy-tooling/integration-tests/alias-test/generate/target/generatedOnline-12.2.1.4.0.210930.json
[DEBUG]   (s) namespaceName = <redacted>
[DEBUG]   (s) objectName = generatedOnline-12.2.1.4.0.210930.json
[DEBUG]   (f) profile = DEFAULT
[DEBUG]   (f) project = MavenProject: com.oracle.weblogic.lifecycle:weblogic-deploy-alias-test-generate:2.3.4-SNAPSHOT @ /Users/rpatrick/Projects/weblogic-deploy-tooling/integration-tests/alias-test/generate/pom.xml
[DEBUG]   (f) showSecrets = false
[DEBUG]   (f) skip = false
[DEBUG] -- end configuration --
```

If I do a thread dump, I see that the code is inside Banner.groovy

```
"main" #1 prio=5 os_prio=31 tid=0x00007fc91480a000 nid=0x1803 runnable [0x00007000016fd000]
   java.lang.Thread.State: RUNNABLE
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:107)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1268)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1035)
	at org.codehaus.groovy.runtime.InvokerHelper.invokePojoMethod(InvokerHelper.java:1017)
	at org.codehaus.groovy.runtime.InvokerHelper.invokeMethod(InvokerHelper.java:1008)
	at org.codehaus.groovy.runtime.typehandling.DefaultTypeTransformation.compareEqual(DefaultTypeTransformation.java:654)
	at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.compareEqual(ScriptBytecodeAdapter.java:708)
	at org.kordamp.maven.plugin.oci.Banner.display(Banner.groovy:50)
	at org.kordamp.maven.plugin.oci.mojos.AbstractOCIMojo.execute(AbstractOCIMojo.groovy:127)
	at org.apache.maven.plugin.DefaultBuildPluginManager.executeMojo(DefaultBuildPluginManager.java:137)
...
```

In looking at the source, it seems clear that the `while(true)` loop is not correct in that the `element` variable used to control the `break` is never changed inside the loop (so it either breaks the first time through or never).  See https://github.com/kordamp/oci-maven-plugin/blob/master/oci-maven-plugin/src/main/groovy/org/kordamp/maven/plugin/oci/Banner.groovy#L49-L54

**Comment by robertpatrick** [comment]
@aalmiray if I make this change to Banner.groovy, the code seems to work properly in a multi-module project.

```
rpatrick@rpatrick-mac oci-maven-plugin % git diff
diff --git a/oci-maven-plugin/src/main/groovy/org/kordamp/maven/plugin/oci/Banner.groovy b/oci-maven-plugin/src/main/groovy/org/kordamp/maven/plugin/oci/Banner.groovy
index ade5309..2feae71 100644
--- a/oci-maven-plugin/src/main/groovy/org/kordamp/maven/plugin/oci/Banner.groovy
+++ b/oci-maven-plugin/src/main/groovy/org/kordamp/maven/plugin/oci/Banner.groovy
@@ -44,13 +44,12 @@ final class Banner {
     }
 
     static void display(MavenProject project, Log log) {
-        MavenProject element = project
         MavenProject root = project
         while (true) {
-            if (element.parent == null || element.parent == element) {
+            if (root.parent == null || root.parent == root) {
                 break
             }
-            root = element.parent
+            root = root.parent
         }
         if (b.visited.contains(root.name)) {
             return
rpatrick@rpatrick-mac oci-maven-plugin % 
```

**Comment by aalmiray** [comment]
Fixed in `0.5.1`
